api/src/main/java/io/knotx/engine/api/KnotFlowStep.java

```diff
@@ -0,0 +1,119 @@
+/*
+ * Copyright (C) 2019 Knot.x Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.knotx.engine.api;
+
+import io.vertx.circuitbreaker.CircuitBreakerOptions;
+import io.vertx.codegen.annotations.DataObject;
+import io.vertx.core.eventbus.DeliveryOptions;
+import io.vertx.core.json.JsonObject;
+import java.util.Objects;
+import org.apache.commons.lang3.StringUtils;
+
+@DataObject(generateConverter = true)
+public class KnotFlowStep {
```

Comment:
If we decide to rename `KnotFlow ` to e.g. `FragmentFlowStep`, how about `FlowStepOtions` here?
Then `address` could be `knotAddress`.